docs/design/properties_code_point_trie.md

```diff
@@ -77,10 +107,21 @@ The advantages would be having code in ICU4X that shares the same precomputed op
 
 The disadvantages would be similar to what are applicable for other external sources of data in ICU4X that go through a data provider (an extra data dependency that requires an offline step for downloading/exporting during build and/or installation time).
 
-### Conclusion
+### Decision
 
 The preferred approach is option 2, to implement a reader in Rust of the ICU binary format of `CodePointTrie`. The disadvantages are fairly minimal and not different from those of other external sources of data for ICU4X's data provider. However, the upsides of optimal performance, less code to write, and better integration with ICU are each considered to be significant. 
 
 Choosing this option does not rule out a full implementation of the code point trie builder, but the cost benefit analysis in the meeting notes would need to be revisited for that. For the current constraints, using ICU4C data is a reasonable approach.
 
 Because the binary format of the `CodePointTrie` can vary depending on the endian-ness of the target architecture, it will be easiest to export the bytes as plain text data literals (ex: in TOML format). For more details on the specific representation that will be used within the offline step in ICU4C to dump the code point trie information, see [this design doc](https://docs.google.com/document/d/1JkrL4pv477dIVnfilwlAEwqF9vstb_Uo_5XeUz2WXM8/edit#heading=h.9sqixr5cv2wn) for that topic.
+
+## Building a `CodePointTrie`
+
+Building a `CodePointTrie` is expensive because several optimizations are applied during build time in order to reduce the size with little to no effect on the runtime.
+In ICU, when trie data is built for Unicode properties, it is done in a compile-time step and stored statically, which therefore does not effect runtime performance.
+One example of an optimization is called compaction, in which subarrays (of size 2^6 = 64, called "blocks") which have identical contents can be collapsed and treated as being identical without affecting the trie lookup algorithm's result.
```

Comment:
The size of the data blocks depends on whether it's in the fast or small lookup range.
The compaction is also done on each part of the index array, not just on the data array.
Maybe just remove the parenthesis?

Reply:
Done.